src/components/AdminAddBusiness/AdminAddBusiness.jsx

```diff
@@ -0,0 +1,631 @@
+      });
+      setDonationSiteInformation(prevState => ({
+        ...prevState,
+        nameOfCompany: '',
+        firstName: '',
+        lastName: '',
+        email: '',
+        website: '',
+        streetNumber: '',
+        unitNumber: '',
+        zip: '',
+        state: '',
+        city: '',
+        country: '',
+        phoneAreaCode: '',
+        phoneNumber: '',
+        numberOfHoursWorked: '',
+        howTheyHearAboutUs: '',
+        internalNotes: '',
+        termsAndCondition: false,
+        petFoodProviderSite: false,
+        donationSite: false,
+        shelter: false,
+        domesticViolenceShelter: false,
+        familiesOnlyShelter: false,
+        wellnessClinics: false,
+        spayNeuter: false,
+        financialAssistance: false,
+        rehomeFoster: false,
+        enteredInQB: false,
+        inactive: false,
+        finalCheck: false,
+        erBoarding: false,
+        seniorCitizensOnly: false,
+        cancerSpecific: false,
+        dogSpecific: false,
+        catSpecific: false,
+      }));
+      clearInputField();
+    } catch (error) {
+      console.error('Error updating data:', error);
+    }
+    return;
+  };
+  /* CHANGE FOR THIS PAGE WHEN WE REACH BACKEND
+  const updateContactInfo = async () => {
+    try {
+      await backend.put(`/business/${BUSINESS_ID}`, {
+        business_name: businessContactInfo.businessName,
+        contact_name: `${businessContactInfo.firstName} ${businessContactInfo.lastName}`,
+        contact_phone: businessContactInfo.phoneNumber,
+        primary_email: businessContactInfo.email,
+        website: businessContactInfo.website,
+        street: businessContactInfo.street,
+        zip: businessContactInfo.zip,
+        city: businessContactInfo.city,
+        state: businessContactInfo.state,
+        business_hours: businessContactInfo.business_hours,
+      });
+    } catch (error) {
+      console.error('Error updating data:', error);
+    }
+
+  };
+
+  useEffect(() => {
+    const getBusinessContactResponse = async () => {
+      try {
+        const businessContactResponse = await backend.get(`/business/${BUSINESS_ID}`);
+        const businessContact = businessContactResponse.data[0];
+        const name = businessContact.contact_name.split(' ');
+        const firstName = name[0];
+        const lastName = name[1];
+
+        setBusinessContactInfo({
+          businessName: businessContact.name,
+          phoneNumber: businessContact.contact_phone,
+          email: businessContact.primary_email,
+          website: businessContact.website,
+          street: businessContact.street,
+          firstName: firstName,
+          lastName: lastName,
+          city: businessContact.city,
+          state: businessContact.state,
+          zip: businessContact.zip_code,
+          business_hours: businessContact.business_hours,
+        });
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
```

Comment:
can you explain this more in detail? not really understanding what the comment is referring to. but delete if we don't need it